### PR TITLE
idea for implementing a rule to process command from start

### PR DESCRIPTION
**Pull request by luwqz1**

### CODE REVIEW

File: telegrinder/bot/rules/__init__.py [hunk 1]
```diff
@@ -12,3 +12,5 @@
 from .text import Text, HasText, ABCTextMessageRule
 from .fuzzy import FuzzyText
 from .integer import Integer, IntegerInRange
+from .start_query import (StartQuery, StrQuery, IntQuery, MarkupQuery, 
+                          ABCQuery)
```
Comment (timoniq) [inline telegrinder/bot/rules/__init__.py:16]: fix formatting pls

File: telegrinder/bot/rules/start_query.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+from .abc import ABC, Message, abstractmethod
+from .markup import PatternLike, check_string
+from .text import ABCTextMessageRule
+import typing
+import vbml
+
+
+class ABCQuery(ABC):
```
Comment (timoniq) [inline telegrinder/bot/rules/start_query.py:8]: bad idea: redundant abstraction, consider using vbml validators

File: telegrinder/bot/rules/start_query.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class StartQuery(ABCTextMessageRule):
+    def __init__(self, query: typing.Optional[ABCQuery] = None) -> None:
+        self.query = query
+    
+    async def check(self, message: Message, ctx: dict) -> bool:
+        if not message.text.startswith("/start "):
+            return False
+        query = message.text[7:]
```
Comment (timoniq) [inline telegrinder/bot/rules/start_query.py:60]: .removeprefix
integers for strings slices look cringy honestly

**Review by timoniq — CHANGES_REQUESTED** [review]

**Comment by timoniq** [comment]
hello!
thank you for your PR, I find the idea good but this will need some improvements to get it merged

File: telegrinder/bot/rules/start_query.py (hunk 3 above)
Comment (luwqz1) [reply telegrinder/bot/rules/start_query.py:60]: I understand... but the fact is that telegrinder supports the version of python 3.8.x and in it str did not have the removeprefix method, it appeared with 3.9.x

Comment (luwqz1) [reply telegrinder/bot/rules/start_query.py:60]: I was able to come up with a solution. i use lstrip method

Comment (timoniq) [reply telegrinder/bot/rules/start_query.py]: > I understand... but the fact is that telegrinder supports the version of python 3.8.x and in it str did not have the removeprefix method, it appeared with 3.9.x

oops i guess i want to switch to 3.11 later, i'll open issue

**Pull request closed without merging**